src/vcpkg/commands.integrate.cpp

```diff
@@ -466,7 +465,7 @@ namespace vcpkg::Commands::Integrate
                        .string_arg("-ExecutionPolicy")
                        .string_arg("Bypass")
                        .string_arg("-Command")
-                       .string_arg(Strings::format("& {& '%s' }", script_path));
+                       .string_arg(fmt::format("& {& '{}' }", script_path));
```

Comment:
Probably the reason for the following crash: https://github.com/microsoft/vcpkg/issues/30283

Reply:
It can solve the problem of `{}` in the parsing path.